torch_glow/src/FuseConcat.cpp

```diff
@@ -0,0 +1,52 @@
+#include "FuseConcat.h"
+namespace glow {
+bool isFusableConcatNode(torch::jit::Node *node) {
```

Comment:
also this isn't used outside of this cpp file so make it static or put in `namespace{}`